src/appmixer/ai/claude/lib.js

```
@@ -0,0 +1,117 @@
+
+        // Create a new assistant with tools defined in the branches connected to my 'tools' output port.
+        const tools = {};
+        let error;
+
+        // Find all components connected to my 'tools' output port.
+        Object.keys(flowDescriptor).forEach((componentId) => {
+            const component = flowDescriptor[componentId];
+            const sources = component.source;
+            Object.keys(sources || {}).forEach((inPort) => {
+                const source = sources[inPort];
+                if (source[agentComponentId] && source[agentComponentId].includes(toolsPort)) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)